common/config/azure-pipelines/templates/core-build.yaml

```diff
@@ -74,7 +74,7 @@ steps:
   - script: node common/scripts/install-run-rush.js audit
     displayName: rush audit
     workingDirectory: ${{ parameters.workingDir }}
-    condition: and(succeeded(), ${{ parameters.runRushAudit }}, ne('${{ parameters.targetBranch }}', 'imodel02'))
+    condition: and(succeeded(), ${{ parameters.runRushAudit }}, ne('${{ parameters.targetBranch }}', 'master'))
```

Comment:
I think `ne('${{ parameters.targetBranch }}', 'master')` can be removed

Reply:
> I think `ne('${{ parameters.targetBranch }}', 'master')` can be removed

I agree. otherwise audit would not run on master which is a required thing.